python/ray/worker.py

```diff
@@ -945,12 +945,14 @@ def _wait_for_and_process_task(self, task):
             }
             if task.actor_id().id() == NIL_ACTOR_ID:
                 title = "ray_worker:{}()".format(function_name)
+                next_title = "ray_worker"
             else:
                 actor = self.actors[task.actor_id().id()]
                 title = "ray_{}:{}()".format(actor.__class__.__name__,
                                              function_name)
+                next_title = "ray_{}".format(actor.__class__.__name__)
             with profiling.profile("task", extra_data=extra_data, worker=self):
-                with _changeproctitle(title):
+                with _changeproctitle(title, next_title):
```

Comment:
Noticed that during A3C testing you sometimes see "ray_worker" for actor processes. This fixes that by explicitly setting the next title instead of it going to back to whatever it was before.